### ISSUE TITLE
Docker error

### ISSUE THREAD

**Issue by kylebgorman** [ElementAI/build-pynini-wheels#2]
Hi, sorry for my belated attempt at integrating this. I just updated the Dockerfile to point to the current versions of OpenFst and Pynini and ran `docker build --target=run-tests -t build-pynini-wheels .` as per instructions. I get the following weird error in the middle of the process:

```
/opt/_internal/cpython-3.7.9/bin/python3.7: error while loading shared libraries: libcrypt.so.2: cannot open shared object file: No such file or directory
The command '/bin/sh -c for PYBIN in /opt/python/*/bin; do     "${PYBIN}/pip" install --upgrade pip -r /src/pynini-${PYNINI_VERSION}/requirements.txt     || exit 1; done' returned a non-zero code: 1
```

**Comment by daanzu** [comment]
@kylebgorman I didn't get that error, but I updated the build in https://github.com/ElementAI/build-pynini-wheels/pull/3.

**Comment by PierreAndreNoel** [comment]
Should now be fixed following #4
